addons/mail/static/src/core/common/suggestion_hook.js

```diff
@@ -86,6 +95,14 @@ export class UseSuggestion {
         let text = "";
         if (this.comp.composerService.htmlEnabled) {
             const selection = this.comp.editor.shared.selection.getEditableSelection();
+            if (
+                !isTextNode(selection.startContainer) ||
+                !isContentEditable(selection.startContainer) ||
+                !selection.isCollapsed
+            ) {
```

Comment:
Something is not quite right here:

<img width="313" height="155" alt="image" src="https://github.com/user-attachments/assets/7be4d13b-1279-4968-bba4-32de04b819c0" />

I'm still able to get the list showing even though it is an already selected mention.